### PR TITLE
created packer for AMI creation

### PR DESCRIPTION
**Pull request by Iliabuleh**
Signed-off-by: Ilia Buleh <ilia@nrgene.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,3 +81,53 @@ Add the following parameters to the nextflow run command:
...
+        "tags": {
+            "Environment": "nrglab", 
+            "Job": "maestro3", 
+            "Task": "maestro3", 
+            "Service": "aws-batch"
+        }, 
+        "spotIamFleetRole": "arn:aws:iam::665945310045:role/GenomicsEnv-Batch-IamStac-GenomicsEnvBatchSpotFlee-RQX330WWT2UV"
+    }
+}
+~~~
```
Comment (doron-st) [inline README.md]: @Iliabuleh can you add instructions for using the packer?

File: packer/nextflowAwsBatch.json.org [hunk 2]
```diff
@@ -0,0 +1,74 @@
+{
+  "variables": {
```
Comment (doron-st) [inline packer/nextflowAwsBatch.json.org]: why do we need two nextflowAwsBatch.json files?

File: README.md [hunk 3]
```diff
@@ -81,3 +81,53 @@ Add the following parameters to the nextflow run command:
...
+
+
+### Creating Compute Environment using Cli:
+~~~
+aws batch create-compute-environment \
+--cli-input-json file://cli.json \
+--type MANAGED \
+--compute-environment-name maestro3-stable-small-machines-modified-kernel \
+--service-role arn:aws:iam::665945310045:role/GenomicsEnv-Batch-IamStac-GenomicsEnvBatchServiceR-13WP0LC1KSMS6
+~~~
```
Comment (doron-st) [inline README.md]: Very cool1

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+manifest.json
```
Comment (noamBarkai) [inline .gitignore]: not needed in source control?

File: packer/nextflowAwsBatch.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    "ssh_private_key_file": "~/.ssh/nrglab.pem"
+  },
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "access_key": "{{user `aws_access_key`}}",
+      "secret_key": "{{user `aws_secret_key`}}",
+      "region": "{{user `vpc_region`}}",
+      "vpc_id": "{{user `vpc_id`}}",
+      "subnet_id": "subnet-7b5f4b50",
```
Comment (noamBarkai) [inline packer/nextflowAwsBatch.json]: how does this integrate with Teraform  if and when we move this to production?

Comment (Iliabuleh) [reply packer/nextflowAwsBatch.json]: @noamBarkai 
You can tag an AMI and then query this AMI ID (using the tag) when trying to create some resource.

See simple example:
https://dzone.com/articles/how-to-deploy-apps-effortlessly-with-packer-and-te

File: .gitignore (hunk 4 above)
Comment (Iliabuleh) [reply .gitignore]: @noamBarkai 
This is the output summery of an AMI build

File: packer/nextflowAwsBatch.json.org (hunk 2 above)
Comment (Iliabuleh) [reply packer/nextflowAwsBatch.json.org]: @doron-st 
We don't.
One of them has EBS management example.
Thought maybe we keep it for reference..

**Pull request merged**